node/narwhal/src/helpers/storage.rs

```diff
@@ -602,22 +602,22 @@ mod tests {
     /// Asserts that the storage matches the expected layout.
     pub fn assert_storage<N: Network>(
         storage: &Storage<N>,
-        committees: Vec<(u64, Committee<N>)>,
-        rounds: Vec<(u64, IndexSet<(Field<N>, Field<N>, Address<N>)>)>,
-        certificates: Vec<(Field<N>, BatchCertificate<N>)>,
-        batch_ids: Vec<(Field<N>, u64)>,
-        transmissions: HashMap<TransmissionID<N>, (Transmission<N>, IndexSet<Field<N>>)>,
+        committees: &Vec<(u64, Committee<N>)>,
+        rounds: &Vec<(u64, IndexSet<(Field<N>, Field<N>, Address<N>)>)>,
+        certificates: &Vec<(Field<N>, BatchCertificate<N>)>,
+        batch_ids: &Vec<(Field<N>, u64)>,
+        transmissions: &HashMap<TransmissionID<N>, (Transmission<N>, IndexSet<Field<N>>)>,
     ) {
         // Ensure the committees are well-formed.
-        assert_eq!(storage.committees_iter().collect::<Vec<_>>(), committees);
+        assert_eq!(storage.committees_iter().collect::<Vec<_>>(), *committees);
```

Comment:
```suggestion
        assert_eq!(storage.committees_iter().collect(), *committees);
```
applies to others as well

Reply:
Compiler complains about "type annotations needed" if I remove them :disappointed:

Reply:
ah, it's possible with the change from `&Vec<T>` to `&[T]`; nvm then :ok_hand: